extensions/amp-story/1.0/amp-story-scroll.css

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+amp-story[scroll] {
+  contain: none !important;
+}
+
+[scroll] amp-story-page {
+  position: relative !important;
+  height: 100vh !important;
+}
+
+amp-story[scroll] > amp-story-page.i-amphtml-layout-container[distance] {
```

Comment:
I assume the `.i-amphtml-layout-container[distance]` is for precedence; we should probably look to resolve that another way long term, but in the short term, we should probably add a comment.

Reply:
100% agreed
Filed: #17170